example/zend-3.0.3/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM php:7.0-apache
```

Comment:
If this is an easier way of running the example without MAMP, can we add something in the README.md about running it in docker?